packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -382,36 +388,31 @@ const InternalDashboard: React.FC<InternalDashboardProperties> = ({
 
   return (
     <I18nProvider locale='en' messages={[messages]}>
-      <TimeSync
```

Comment:
no change here apart from moving the TimeSync component up one level to the `Dashboard` component